src/components/sidebar/LineItems.tsx

```diff
@@ -9,23 +9,16 @@ import * as s from './lineItems.scss';
 
 interface ILineItemsProps {
     lineStore?: LineStore;
-    searchInput: string;
+    location: any;
```

Comment:
We should not push location prop down all the way through the tree of components. Overall, I don't think LineItem should need to worry about location stuff at all.

Reply:
I've used it in somecases as a workaround for issues with mobx and react-router. It's helped in cases where site isn't getting updated when url changes.

Reply:
That said, I agree on having less information pushed to the furthest components and maybe this can be resolved.

Reply:
In this case it's for the LineItem to be able to form urls for displaying chosen routes.